TodoWithGenericControllers/Program.cs

```diff
@@ -0,0 +1,42 @@
+#define USE_EF
+
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+#if USE_EF
+using Microsoft.EntityFrameworkCore;
+#endif
+
+namespace Todos
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+#if USE_EF
```

Comment:
Why doesn't the package cover up some of this uglyniess with extension methods?

Why is there an EF ifdef?

Reply:
The purpose of that is to switch with EF or without. Also it remove unused using on file when we switch.
![davidfowl_todo_define](https://user-images.githubusercontent.com/309413/74963618-9c005c00-5444-11ea-99fa-c1c38f4a59d1.png)

or should remove it
or new project (this is dilemma since this is repo purpose to show simple ASP.NET Core Todos implementation.

Reply:
Remove it.